### ISSUE TITLE
[Adaptive Cards - Designer - ELEMENT PROPERTIES] - Focus order is not logical on the page on after activating any menu item present inside the 'Action', 'Action type' and 'Add an action' expandable control on the page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9328]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Designer' control present on the header section of the page using tab key and activate using enter key.
3. Navigate to the 'ELEMENT PROPERTIES' flyout using tab key and activate using enter key.
4. Navigate to the 'Action', 'Action type' and 'Add an action' expandable control using tab key and activate using enter key.
5. Now activating any of the menu item using enter key.
6. Verify that, focus order is logical on the page on after activating any menu item present inside the 'Action', 'Action type' and 'Add an action' expandable control on the page or not.

**Actual Result:**
Focus order is not logical on the page on after activating any menu item present inside the 'Action', 'Action type' and 'Add an action' expandable control on the page. Currently on after activating the menu item from 'Action' and 'Action type' control focus gets lost and on pressing next Tab key press focus lands on the 'Data context' edit field while in 'Add an action' expandable control focus lands on the 'Adaptive Card editor' section due to which users face difficulty on again navigating to the triggered control.

**Reproducibility:**
Same issue replicates that focus order is not logical on the page on after closing the 'Add an action' expandable control using esc key. **Refer Attachment:** Reproducibility_FocusOrder

**Observation:**
Currently user have to press 20 times Tab keystrokes to reach the desired control and while in backward navigation user have to press 80 times Shift + Tab keystrokes.

**Expected Result:**
Focus order should be logical on the page on after activating any menu item present inside the 'Action', 'Action type' and 'Add an action' expandable control on the page.  Currently on after activating the control focus should not get lost and it should directly return back to the same triggered interactive control i.e., combo box control.

**User Experience:**
Keyboard dependent users will be impacted if the focus order is not logical on the page on after activating any menu item present inside the 'Action', 'Action type' and 'Add an action' expandable control on the page. Due to which users will not be able to perform the required action as it initiates the incorrect flow and user again need to perform extra keystrokes navigate to the required control and might get confused in terms of identifying the focus order of the page.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/e53b5ad0-6aae-4611-ade2-67d49c93d390)

https://github.com/user-attachments/assets/224b431b-6f74-446a-98d1-7c1435a184a0

**Reproducibility_FocusOrder:**

https://github.com/user-attachments/assets/f7afce56-1b51-4eb0-a5bb-d6e57e0da93d

### Card JSON

```JSON
NA
```
